spec/lib/inventory_dashboard_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+require 'inventory_dashboard.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.